pkg/daemon/update.go

```diff
@@ -702,6 +691,36 @@ func (dn *Daemon) updateOS(config *mcfgv1.MachineConfig) error {
 	return nil
 }
 
+func (dn *Daemon) readPendingConfig() (string, error) {
+	// TODO(runcom): msgid has been generated with journalctl --new-id128, move it to a const
+	journalOutput, err := exec.Command("journalctl", "-b", "-1", "-o", "cat", "MESSAGE_ID=34c7912c5dd2454286097f8f92a22e9e").CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	lines := strings.Split(string(journalOutput), "\n")
+	if len(journalOutput) > 0 {
+		return lines[len(lines)-1], nil
+	}
+	return "", fmt.Errorf("no pending config found in journal: %v", string(journalOutput))
+}
+
+func (dn *Daemon) logPendingConfig(pending *mcfgv1.MachineConfig) error {
+	logger := exec.Command("logger", "--journald")
+	stdin, err := logger.StdinPipe()
+	if err != nil {
+		return fmt.Errorf("failed to get stdin pipe: %v", err)
+	}
+
+	pendingConfigStr := fmt.Sprintf(`MESSAGE_ID=34c7912c5dd2454286097f8f92a22e9e
```

Comment:
This could use some abstraction into a wrapper that takes `key=value` strings as varargs or so.

(One other random thought I had is that we `systemd-run` a service on the host and send our logs to it and have it log them; then we can properly `systemctl status openshift-machine-config` or so - it would also basically act as a host-side mutex too to make doubly sure there aren't multiple MCDs)